DigitalLearningSolutions.Data/Helpers/ConfigHelper.cs

```diff
@@ -34,5 +39,20 @@ public static bool IsSignpostingUsed(this IConfiguration config)
         {
             return bool.Parse(config[UseSignposting]);
         }
+
+        public static int GetLearningHubSsoHashTolerance(this IConfiguration config)
+        {
+            return int.Parse(config[string.Join(':', LearningHubSsoSectionKey, LearningHubSsoToleranceKey)]);

```

Comment:
For these string concatenations the common approach now is to use string interpolation, e.g.
`return int.Parse(config[$"{LearningHubSsoSectionKey}:{LearningHubSsoToleranceKey}"]);`
Could these be changes to use the string interpolation approach?